### ISSUE TITLE
Expose hashrate

### ISSUE THREAD

**Issue by sborrazas** [aeternity/ae_mdw#753]
refs #124 

@marc0olo I'm assuming this hashrate would be per-generation, would that be right?

If so, this can simply be an attribute added on the `/v2/deltastats` endpoint for each generation

**Comment by marc0olo** [comment]
> @marc0olo I'm assuming this hashrate would be per-generation, would that be right?

if possible it would be awesome to have the history about the hashrate per-generation, yeah!

**Comment by thepiwo** [comment]
yes, the node doesn't have this information either, we would need to calculate it from target
